### ISSUE TITLE
Make flush method not shutdown

### ISSUE THREAD

**Issue by rjurney** [kislyuk/watchtower#38]
I need my messages to flush after each send, but your flush method shuts down. Can you change this?

**Comment by kislyuk** [comment]
As a quick workaround for flush not doing what you want, you can use `use_queues=False` when constructing the log handler to make each log message trigger a log delivery API call.

**Comment by kislyuk** [comment]
I agree in general that the semantics of flush() as specified by https://docs.python.org/3/library/logging.html#logging.Handler.flush mean it should not trigger shutdown by default.

**Comment by terencehonles** [comment]
@kislyuk do you know if this is still the case? It looks like it's not since flush looks like it just puts a sentinel on the queue and will force the queues to send when they send it but they continue monitoring the queue after that:

https://github.com/kislyuk/watchtower/blob/5002639093154d29fdeb1bbeb8a2fb30c3b7f48e/watchtower/__init__.py#L286-L297

and

https://github.com/kislyuk/watchtower/blob/5002639093154d29fdeb1bbeb8a2fb30c3b7f48e/watchtower/__init__.py#L256-L257
